config/initializers/git_sha.rb

```diff
@@ -0,0 +1,26 @@
+revisions_logfile = "/opt/alexandria/revisions.log"
```

Comment:
This path should probably be defined in `config/application.yml` since that's where we specify most (ideally all) of our server-specific paths.

Reply:
If you haven't modified the default capistrano directory structure (http://capistranorb.com/documentation/getting-started/structure/) the `revisions.log` file will always be two directories above the application root, so you don't really need to explicitly define a path for this, you could instead infer it:
```
revisions_logfile = File.join( File.expand_path('..//..', Rails.root), 'revisions.log' )
```
There's probably even something more elegant than that, but the main point is we can infer the location instead of having to explicitly set it, which might make setup simpler.